app/controllers/registrations_controller.rb

```diff
@@ -1,3 +1,28 @@
 class RegistrationsController < Devise::RegistrationsController
   respond_to :json
+  prepend_before_action :create, only: [:create]
+  
+  def create
```

Comment:
Did you manually test all this?

Reply:
Yes. If you fail the captcha, you get a notice of failure and no user is created (and the form clears out). If you pass the captcha, you get a notice to confirm your account and a user is created.